webapp/src/client.ts

```diff
@@ -5,8 +5,7 @@ import type {EmojiData, CallsClientJoinData} from '@mattermost/calls-common/lib/
 
 import {EventEmitter} from 'events';
 
-// @ts-ignore
-import {deflate} from 'pako/lib/deflate';
+import {zlibSync, strToU8} from 'fflate';
```

Comment:
My own interest, why this change?

Reply:
@cpoile It's ~40% faster, proper typescript and 5 times smaller bundle size. It got me convinced.